hariharanBala-CBT/strongr-web
# fix: issue 4 in #56

Ticket #56:

QA History:
-> QA notified of the issue in organization page. "Getting yellow page issue on ADD location page, when we add exceed number in phone number , it should a toast message". On investigation the issue is related to permission of different groups in the database.

Steps Taken:
-> Added a Groupmixin in both class based and function based views of organization.
-> Added a custom 403 page.
![image](https://github.com/user-attachments/assets/5aa1912d-a4e7-4506-a316-c59694c739fa)


other issue fixed:
-> Some pages in tenant login displays yellow page issue on load. Fixed confirmed and cancelled organization list page on the tenant side.
-> Fixed Issue 3 in ticket#51.
